### PR TITLE
Update CompatibilitySuppressions.xml after sorting

### PR DESCRIPTION
**Pull request by smasher164**

### CODE REVIEW

**Review by ViktorHofer — CHANGES_REQUESTED** [review]
Please remove new files and new suppressions. I think some of the new suppressions are false positives and might need a bugfix. Let's scope this down to only apply the sorting change.

File: src/libraries/System.Security.Cryptography.Pkcs/src/CompatibilitySuppressions.xml [hunk 1]
```diff
@@ -102,4 +102,106 @@
     <Left>lib/netstandard2.0/System.Security.Cryptography.Pkcs.dll</Left>
     <Right>lib/net462/System.Security.Cryptography.Pkcs.dll</Right>
   </Suppression>
+  <Suppression>
```
Comment (ViktorHofer) [inline src/libraries/System.Security.Cryptography.Pkcs/src/CompatibilitySuppressions.xml:105]: These are new suppressions. Move them out?

File: src/libraries/System.Diagnostics.EventLog/src/CompatibilitySuppressions.xml [hunk 2]
```diff
@@ -6,4 +6,16 @@
...
+    <Target>System.Diagnostics.EventLog.Messages, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>left</Left>
+    <Right>runtimes/win/lib/net6.0/System.Diagnostics.EventLog.Messages.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0004</DiagnosticId>
+    <Target>System.Diagnostics.EventLog.Messages, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>left</Left>
+    <Right>runtimes/win/lib/net7.0/System.Diagnostics.EventLog.Messages.dll</Right>
+  </Suppression>
```
Comment (ViktorHofer) [inline src/libraries/System.Diagnostics.EventLog/src/CompatibilitySuppressions.xml:20]: Also new suppressions

File: src/libraries/System.Security.Cryptography.Pkcs/src/CompatibilitySuppressions.xml (hunk 1 above)
Comment (smasher164) [reply src/libraries/System.Security.Cryptography.Pkcs/src/CompatibilitySuppressions.xml:105]: ah my bad, i thought you meant new files

**Review by ViktorHofer — APPROVED** [review]
Thanks :)

**Pull request merged**